spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-sidecar/src/main/java/com/alibaba/cloud/sidecar/consul/SidecarConsulAutoRegistration.java

```diff
@@ -59,7 +59,7 @@ public static ConsulAutoRegistration registration(
 			service.setAddress(sidecarProperties.getIp());
 		}
 		service.setName(normalizeForDns(appName));
-		service.setTags(createTags(properties));
+		service.setTags(properties.getTags());
```

Comment:
In Spring Cloud 2020, Consul tags and metadata logic has been changed, please pay attension